source/sass/pages/ways-to-give-page.scss

```diff
@@ -0,0 +1,42 @@
+.app--ways-to-give {
+  .app__header {
+    background: $color--black;
+    color: $color--white;
+    height: 300px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    .button {
+      display: inline-block;
+      text-transform: uppercase;
+    }
+  }
+
+  .container {
+    padding: 2em 10px;
+    max-width: 960px;
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  h1 {
```

Comment:
I was hoping I didn't have to override existing typography rules but without these overrides the Ways to Give page 
lacks of visual hierarchy. 

![Screen Shot 2020-01-17 at 17 20 06](https://user-images.githubusercontent.com/2896608/72656207-964bbe00-394e-11ea-9fbf-3724739a72b8.png)

Reply:
There's too broad of a selector somewhere- notice the uppercase/bulleting of the footer items in the screenshot, and the "More about us" text on the review app is way too big.

Reply:
This isn't what my PR looks like. This is what my PR would look like if I didn't add the font style overrides.

Reply:
I'll add some styling tweaks.